vortex-expr/src/exprs/literal.rs

```diff
@@ -98,9 +97,21 @@ impl LiteralExpr {
     }
 }
 
-impl Display for LiteralExpr {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{}", self.value)
+impl DisplayAs for LiteralExpr {
+    fn fmt_as(&self, df: DisplayFormat, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        match df {
+            DisplayFormat::Compact => {
+                write!(f, "{}", self.value)
+            }
+            DisplayFormat::Tree => {
+                write!(
+                    f,
+                    "LiteralExpr(value: {}, dtype: {})",
```

Comment:
lit()?

Reply:
I was considering dropping the `Expr` at the end, but in general I'm in favor of fully spelled out names here.